project.ts

```diff
@@ -1,8 +1,6 @@
 import { Project } from "slack-cloud-sdk/mod.ts";
 import { ReceiveMetadataEventWorkflow } from "./workflows/receive_metadata_event.ts";
```

Comment:
@stevengill I think we can remove this workflow import as well? This sample app has 2 workflows.

Reply:
nice catch!

Reply:
@mwbrooks fixed

Reply:
👌🏻